scripts/init.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+main ()
+{
+    pip install -r local-requirements.txt &&
+    python -m src.history.build
+}
+
+# Create virtual environment
+python -m venv venv
```

Comment:
venvs are managed in manners beyond the usual venv module, with python versioning and virtual environment managers such as [Poetry](https://python-poetry.org/), [PDM](https://github.com/pdm-project/pdm), [Pipenv](https://pipenv.pypa.io/en/latest/), [Anaconda](https://www.anaconda.com/), [Pyenv](https://github.com/pyenv/pyenv), [Virtualenvwrapper](https://virtualenvwrapper.readthedocs.io/en/latest/), [Hatch](https://hatch.pypa.io/latest/).
Nonetheless this is a good starting point for the less intense python developers of the project.

Reply:
added a note about it in `CONTRIBUTING.md`